IBAnimatable/SlideAnimator.swift

```diff
@@ -5,18 +5,18 @@
 
 import UIKit
 
-public class SlideAnimator: NSObject, AnimatedTransitioning {
+open class SlideAnimator: NSObject, AnimatedTransitioning {
```

Comment:
Override an animator would be even a bad idea. If we need to customise an animator, we either have to add more parameters, otherwise that means it should be a new one. So I'm joining to say that it should be `public` instead of `open`